cordova-react/merges/android/scripts/platformOverrides.js

```diff
@@ -0,0 +1,10 @@
+(function () {
```

Comment:
No, it's a polyfill for old android versions. We can even remove it if you want.

however the windows one is important to allow dynamic scripts on Win8.1 (MSAppHost 2.0)